<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KdedForm</class>
 <widget class="QWidget" name="KdedForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title:window">Enable/Disable Touchpad</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="configureShortcutsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Configure…</string>
        </property>
        <property name="icon">
         <iconset theme="configure">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Keyboard shortcuts:</string>
        </property>
        <property name="buddy">
         <cstring>configureShortcutsButton</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="kcfg_DisableOnKeyboardActivity">
     <property name="title">
      <string>Disable touchpad when typing</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Timeout:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_KeyboardActivityTimeoutMs</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_KeyboardActivityTimeoutMs">
        <property name="toolTip">
         <string>If there was no keyboard activity for more than specified
number of milliseconds, touchpad will be enabled again</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="kcfg_OnlyDisableTapAndScrollOnKeyboardActivity">
        <property name="toolTip">
         <string>Disable click emulation and scrolling.
You will be still able to move pointer using touchpad and perform clicks with hardware buttons.</string>
        </property>
        <property name="text">
         <string>Disable taps and scrolling only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="kcfg_DisableWhenMousePluggedIn">
     <property name="title">
      <string>Disable touchpad when mouse is plugged in</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="notificationsLabel">
          <property name="text">
           <string>Notifications:</string>
          </property>
          <property name="buddy">
           <cstring>configureNotificationsButton</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="configureNotificationsButton">
          <property name="text">
           <string>Configure…</string>
          </property>
          <property name="icon">
           <iconset theme="configure"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="ignoredDevLabel">
        <property name="text">
         <string>Ignored devices:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_MouseBlacklist</cstring>
        </property>
       </widget>
       </item>
       <item>
       <widget class="KEditListWidget" name="kcfg_MouseBlacklist">
        <property name="toolTip">
         <string>Some devices might be incorrectly treated as a mouse.
You can add them to this list so that they will be ignored when they're plugged in.</string>
        </property>
        <property name="buttons">
         <set>KEditListWidget::Add|KEditListWidget::Remove</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QGroupBox</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QSpinBox</class>
   <extends>QWidget</extends>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>configureShortcutsButton</tabstop>
  <tabstop>kcfg_DisableOnKeyboardActivity</tabstop>
  <tabstop>kcfg_KeyboardActivityTimeoutMs</tabstop>
  <tabstop>kcfg_OnlyDisableTapAndScrollOnKeyboardActivity</tabstop>
  <tabstop>kcfg_DisableWhenMousePluggedIn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
